packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -384,11 +391,78 @@ describe('Ceramic interop: core <> http-client', () => {
       fetchJson(`http://localhost:${daemon.port}/api/v0/streams/${doc.id}/content`, {
         timeout: 1000,
       })
-    ).rejects.toThrow(`Http request timed out after 1000 ms`)
+    ).rejects.toThrow(/aborted/)
 
     clearTimeout(id)
   })
 
+  it('Aborts fetch through passed in AbortSignal', async () => {
+    const content1 = { test: 123 }
+    const doc = await TileDocument.create(core, content1, null, { anchor: false })
+
+    const loadStreamMock = jest.spyOn(core, 'loadStream')
+    let id = null
+    loadStreamMock.mockImplementation(() => {
+      return new Promise((resolve) => {
+        id = setTimeout(() => {
+          resolve(doc)
+        }, 2000)
```

Comment:
maybe bump this up to 5 seconds so there's no chance of a race with the timedAbortSignal